src/Containers/App/App.js

```diff
@@ -16,7 +17,7 @@ function App() {
         }} />
         <Route path='/doctors' render={() => {
           return(
-            <h1>Doctors go here</h1>
+            <Doctors />
```

Comment:
I like the set up of the dr's and pros's here for our current render of the app.

Reply:
As well, I prefer our links on all the pages, as you have them already.  If we need to make minor adjustments from here, we can.